### DISCUSSION TITLE
CW-U mode RST defaults to 59 instead of 599

### DISCUSSION THREAD

**Discussion by spREGR** [wavelog/wavelog#686] [Q&A] [answered]
Hi 
I am using Wavelog with Wavelog-Gate which pickups the mode from my FT991-A as CW-U. On the Live QSO screen, the default RST is shown is 59 and not 599. Where do I change the setting so that it defaults to 599.
73
de VU2TUM
![image](https://github.com/user-attachments/assets/91990a9e-5490-4f8c-83cf-b2a99e6d7d58)

**Comment by HB9HIL** [comment]
Hello @spREGR 

You added the mode CW-U by hand right?

**Reply by spREGR** [reply]
Yes, otherwise the Mode is blank by default.

**Reply by HB9HIL** [reply]
roger, currently I don't have a FT-991A here. Is it the same with CW-L?

**Reply by spREGR** [reply]
> roger, currently I don't have a FT-991A here. Is it the same with CW-L?
> 
> 

Yes, same with CW-L

**Comment by HB9HIL** [comment, marked as answer]
Created issue #687, Fix in #688
